playground/src/editor.tsx

```diff
@@ -147,6 +172,27 @@ export function Editor(props: {
     };
   }, []);
 
+  useEffect(() => {
+    function onDiagnostics(evt: LanguageServiceEvent) {
+      const diagnostics = evt.detail.diagnostics;
+      errMarks.current.checkDiags = diagnostics;
+      markErrors(evt.detail.version);
+      setHasCheckErrors(diagnostics.length > 0);
+    }
+
+    props.languageService.addEventListener("diagnostics", onDiagnostics);
+
+    return () => {
+      log.info("Removing diagnostics listener");
+      props.languageService.removeEventListener("diagnostics", onDiagnostics);
+    };
+  }, [props.languageService]);
+
+  useEffect(() => {
```

Comment:
How does this differ from line 211?

Reply:
Argh, thank you. I think the code got duplicated in a merge.